### PR TITLE
Add installation and development setup instructions

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,29 @@ Those groups of users (or circles) can then be used by any other app for sharing
 
 # Installation
 
-(to be written)
+The app is distributed through the [app store](https://apps.nextcloud.com/apps/circles) and you can install it [right from your Nextcloud installation](https://docs.nextcloud.com/server/stable/admin_manual/apps_management.html).
```
Comment (ChristophWurst) [inline README.md:25]: starting with 22 the app is *shipped* with nextcloud server. it is only available via the app store for nextcloud 21 and older

File: README.md [hunk 2]
```diff
@@ -22,7 +22,29 @@ Those groups of users (or circles) can then be used by any other app for sharing
...
+
+Release tarballs are hosted at https://github.com/nextcloud-releases/mail/releases.
+
+## Development setup
+
+Just clone this repo into your apps directory ([Nextcloud server](https://github.com/nextcloud/server#running-master-checkouts) installation needed). Additionally, [npm](https://www.npmjs.com/) to fetch [Node.js](https://nodejs.org/en/download/package-manager/) is needed for installing JavaScript dependencies
```
Comment (ChristophWurst) [inline README.md:31]: I don't think node nor npm are needed

**Comment by ArtificialOwl** [comment]
Not even sure npm/node.js are needed as Circles is mostly back-end now.

beside that, we can merge

**Comment by JuliaKirschenheuter** [comment]
not relevant for now

**Pull request closed without merging**